recipes/libnl/libnl.inc

```diff
@@ -16,7 +16,7 @@ FILES_${PN}-dev += "${libdir}/libnl/cli/*/*.la"
 FILES_${PN}-dbg += "${libdir}/libnl/cli/*/.debug"
 
 inherit auto-package-libs
-AUTO_PACKAGE_LIBS = "nl-3 nl-route-3 nl-nf-3 nl-genl-3 nl-cli-3"
+AUTO_PACKAGE_LIBS = "nl-3 nl-route-3 nl-nf-3 nl-genl-3 nl-cli-3 nl-idiag-3 nl-xfrm-3"
```

Comment:
Adding these in the .inc file, doesn't seem right, as they are then added for old versions when the new version is cherry-picked, which does not make sense.  And the whole purpose of using an .inc file, and this "Add" + "Remove" commit principle, is to be able to cherry-pick without breaking stuff.